ui/app/components/auth/mounts-display.hbs

```diff
@@ -0,0 +1,33 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{#if (gt @mounts.length 1)}}
+  {{! render dropdown of mount paths }}
+  <Hds::Form::Select::Field name="path" data-test-select="path" as |F|>
+    <F.Label>Mount path</F.Label>
+    <F.Options>
+      {{#each @mounts as |mount|}}
+        <option value={{mount.path}}>{{mount.path}}</option>
+      {{/each}}
+    </F.Options>
+  </Hds::Form::Select::Field>
+{{else}}
+  {{! render a single mount path }}
+  {{#let (get @mounts "0") as |mount|}}
+    {{#unless @hideType}}
```

Comment:
This component doesn't always render within the login form tabs, so sometimes we'll want to render the type here